firstvoices/backend/search/documents/song_document.py

```diff
@@ -15,5 +15,10 @@ class SongDocument(BaseDocument):
     note = Text(copy_to="other_translation_search_fields")
     acknowledgement = Text(copy_to="other_translation_search_fields")
 
+    # filter and sorting
+    hasAudio = Keyword()
```

Comment:
(Optional) should we move these three attributes to a separate class that can be inherited (mixin sort of) ? Right now there are these 3 fields, but I guess more can be added for reporting/filtering purposes. The name of the mixin can be something related to media filtering or reporting